pandas/core/indexing.py

```diff
@@ -1648,8 +1649,35 @@ class _iLocIndexer(_LocationIndexer):
     out-of-bounds, except *slice* indexers which allow out-of-bounds
     indexing (this conforms with python/numpy *slice* semantics).
 
-    See more at :ref:`Selection by Position <indexing.integer>`
+    See Also
+    --------
+    DataFrame.ix : A primarily label-location based indexer
+    DataFrame.loc : Fast integer location scalar accessor.
```

Comment:
I think you may want the " A primarily label-location based indexer" for the .loc description.

You may be thinking of `DataFrame.iat` for this dsecription.

Reply:
Remove ix and update loc description:

-    DataFrame.ix : A primarily label-location based indexer
-    DataFrame.loc : Fast integer location scalar accessor.
+    DataFrame.iat : Fast integer location scalar accessor.
+    DataFrame.loc : Purely label-location based indexer for selection by label.

Reply:
missed your comment about .loc .. but instead got the description from https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.loc.html